tasks/cea-install.js

```diff
@@ -146,12 +146,12 @@ const run = async () => {
     console.log('Preparing project directory...');
     await checkTargetDir();
 
-    console.log('Initializing npm dependencies...');
-    npmInit();
-
     console.log('Copying project files...');
     await srcInit();
 
+    console.log('Initializing npm dependencies...');
+    npmInit();
```

Comment:
@hutchgrant 
At this point, since we have the _template/_ now, and _package.json_ isn't shared anymore, I wonder if we all `npmInit` needs to do now is just set the `name` field now?

Reply:
refactored `npmInit`

Reply:
If you'd prefer, but it also gets rid of all the unnecessary fields and just leaves the `depedencies`, `devdependencies`, `private` and `name` fields. Getting rid of the `author`, `license`, `homepage`, `bin`. It's probably better to keep it how it is.

Reply:
@hutchgrant 
Since _package.json_ isn't shared anymore, I just deleted all that extra stuff from _package.json_ in _template/_ anyway.

Basically, now we get the same output, but the only thing we have to change now is the `name` field.